test/provision/k8s_install.sh

```diff
@@ -308,7 +322,7 @@ fi
 #Install kubernetes
 set +e
 case $K8S_VERSION in
-    "1.16"|"1.17"|"1.18"|"1.19"|"1.20"|"1.21")
+    "1.16"|"1.17"|"1.18"|"1.19"|"1.20"|"1.21"|"1.22")
```

Comment:
@aanm Do we drop the oldest version now?

Reply:
Not now. Maybe in the future.